### PR TITLE
Feature/cucumber

### PR DESCRIPTION
**Pull request by rodikno**
- Added a Cucumber specific structure to the project
- Covered all tests from 'old' project with a Cucumber scenarios

### CODE REVIEW

File: features/redmine.feature [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Feature: Redmine project tests
```
Comment (d-konovalov) [inline features/redmine.feature:1]: The only I would suggest here is to add a Background section with ```I register a user``` step

File: features/step_definitions/redmine_steps.rb [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+When(/^I submit the registration form$/) do
+  register_user(@user)
+end
+
+Then(/^New user is registered$/) do
+  success_message = find_element_by_id('flash_notice')
+
+  expect(@driver.current_url).to eql @my_account_page_url
+  expect(success_message.displayed?).to be
```
Comment (d-konovalov) [inline features/step_definitions/redmine_steps.rb]: be_displayed ? :)

File: features/redmine.feature (hunk 1 above)
Comment (rodikno) [reply features/redmine.feature:1]: Yep, thought about it but was confused the `Registration positive` scenario will look like
```ruby
  Scenario: Registration positive
    Then New user is registered
```

But anyways it's more clear now, fixed =)

File: features/step_definitions/redmine_steps.rb (hunk 2 above)
Comment (rodikno) [reply features/step_definitions/redmine_steps.rb]: Yep, missed these in .steps file, fixed.

**Comment by rodikno** [comment]
@d-konovalov also fixed merge conflict in the last commit

**Pull request merged**